### PR TITLE
Fix launcher plugins

### PR DESCRIPTION
**Pull request by vitvakatu**
Enable supervisor service by default
Fix generate-sources for core module to generate correct python module
Rename plugins directory to exonum_launcher_java_plugins

## Overview

---
See: https://jira.bf.local/browse/ECR-XYZ

### Definition of Done

- [ ] There are no TODOs left in the code
- [ ] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [ ] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [ ] Public API has Javadoc
- [ ] Method preconditions are checked and documented in the Javadoc of the method
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

### CODE REVIEW

File: exonum-java-binding/core/pom.xml [hunk 1]
```diff
@@ -41,7 +41,7 @@
     <!-- This flag is empty for debug builds and equals to `release` otherwise -->
     <build.cargoFlag></build.cargoFlag>
     <protobuf.python.outputDirectory>
-      ${project.parent.basedir}/launcher-plugin/exonum_java_runtime_plugin/proto
+      ${project.parent.basedir}/exonum_launcher_java_plugins/exonum_java_runtime_plugin/proto
```
Comment (dmitry-timofeev) [inline exonum-java-binding/core/pom.xml]: Why underscores when all the other use dashes?

File: exonum-java-binding/core/rust/exonum-java/Cargo.toml [hunk 2]
```diff
@@ -12,6 +12,7 @@ failure = "0.1.5"
 # in a correct application early, it is currently required in the application
 # because some public APIs in Java permit passing arbitrary pointers (e.g., Snapshot#newInstance).
 java_bindings = { path = "..", features = ["invocation", "resource-manager"] }
+exonum-supervisor = "0.12"
```
Comment (dmitry-timofeev) [inline exonum-java-binding/core/rust/exonum-java/Cargo.toml]: sorted

File: exonum-java-binding/launcher-plugin/.gitignore [hunk 3]
```diff
@@ -1,2 +0,0 @@
-# Auto-generated sources for proto messages
-exonum_java_runtime_plugin/proto/
```
Comment (dmitry-timofeev) [inline exonum-java-binding/launcher-plugin/.gitignore]: ?

File: exonum-java-binding/core/pom.xml (hunk 1 above)
Comment (vitvakatu) [reply exonum-java-binding/core/pom.xml]: Python naming convention https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html#packages

File: exonum-java-binding/launcher-plugin/.gitignore (hunk 3 above)
Comment (vitvakatu) [reply exonum-java-binding/launcher-plugin/.gitignore]: This file is removed, replaced with proper one

File: exonum-java-binding/exonum_launcher_java_plugins/.gitignore [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Generated protobuf modules
+exonum_java_runtime_plugin/proto
+
+# Python packaging and caches
+*/__pycache__/
+*.egg-info/
```
Comment (dmitry-timofeev) [inline exonum-java-binding/exonum_launcher_java_plugins/.gitignore]: 🍳

**Pull request merged**